<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Veyon Configurator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../veyon-configurator.qrc">
    <normaloff>:/resources/veyon-configurator.png</normaloff>:/resources/veyon-configurator.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="configPages">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="GeneralConfigurationPage" name="generalConfigurationPage"/>
      <widget class="ServiceConfigurationPage" name="serviceConfigurationPage"/>
      <widget class="MasterConfigurationPage" name="masterConfigurationPage"/>
      <widget class="AuthenticationConfigurationPage" name="authenticationConfigurationPage"/>
      <widget class="AccessControlPage" name="accessControlPage"/>
      <widget class="QWidget" name="debugPage">
       <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,1">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>Reporting bugs in Veyon</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,0,0,0,0,0,0">
           <property name="spacing">
            <number>16</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>When encountering bugs or other misbehaviour in Veyon it is important to inform the developers about them, so that the bugs can be fixed in future releases. This page allows you to submit a helpful bug report easily.

If the bug is not reproducible, skip step 1) - 3).</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>1)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Clear the logfiles in the &quot;General&quot; section.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_25">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>2)</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>In the &quot;General&quot; section set the log level to &quot;Debug messages and everything else&quot;.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
             <item>
              <widget class="QLabel" name="label_19">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>3)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Repeat the actions leading to the bug or misbehaviour.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_21">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>4)</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Click the following button and save the file to a known location.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0,1">
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="generateBugReportArchive">
               <property name="text">
                <string>Generate bug report archive</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_14">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_5" rowstretch="1,0" columnstretch="0,1">
             <item row="0" column="0" rowspan="2">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_23">
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>5)</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_24">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Go to the &lt;a href=&quot;https://github.com/Veyon/veyon/issues&quot;&gt;Veyon issue tracker&lt;/a&gt;, create a new issue, describe what you did in step 3) and attach the file you saved in step 4).&lt;/p&gt;
&lt;br/&gt;
&lt;br/&gt;
Note: if the bug involves both a master and client computer, attach bug report archives for both computers.</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QListWidget" name="pageSelector">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="viewMode">
       <enum>QListView::ListMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="currentRow">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>General</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/configure-shortcuts.png</normaloff>:/resources/configure-shortcuts.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Service</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/application-x-sharedlib.png</normaloff>:/resources/application-x-sharedlib.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Master</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/application-x-ms-dos-executable.png</normaloff>:/resources/application-x-ms-dos-executable.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Authentication</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/folder-publicshare.png</normaloff>:/resources/folder-publicshare.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Access control</string>
       </property>
       <property name="icon">
        <iconset resource="../../core/core.qrc">
         <normaloff>:/resources/application-x-pem-key.png</normaloff>:/resources/application-x-pem-key.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bug reporting</string>
       </property>
       <property name="icon">
        <iconset resource="../veyon-configurator.qrc">
         <normaloff>:/resources/tools-report-bug.png</normaloff>:/resources/tools-report-bug.png</iconset>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1294</width>
     <height>35</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadSettings"/>
    <addaction name="actionSaveSettings"/>
    <addaction name="separator"/>
    <addaction name="actionResetConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutVeyon"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="icon">
    <iconset resource="../veyon-configurator.qrc">
     <normaloff>:/resources/document-save.png</normaloff>:/resources/document-save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save settings into file</string>
   </property>
   <property name="toolTip">
    <string>Save settings into file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="icon">
    <iconset resource="../veyon-configurator.qrc">
     <normaloff>:/resources/document-open.png</normaloff>:/resources/document-open.png</iconset>
   </property>
   <property name="text">
    <string>L&amp;oad settings from file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAboutVeyon">
   <property name="icon">
    <iconset resource="../../core/core.qrc">
     <normaloff>:/resources/info.png</normaloff>:/resources/info.png</iconset>
   </property>
   <property name="text">
    <string>About Veyon</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionResetConfiguration">
   <property name="icon">
    <iconset resource="../veyon-configurator.qrc">
     <normaloff>:/resources/edit-delete.png</normaloff>:/resources/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Reset configuration</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccessControlPage</class>
   <extends>QWidget</extends>
   <header>AccessControlPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GeneralConfigurationPage</class>
   <extends>QWidget</extends>
   <header>GeneralConfigurationPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MasterConfigurationPage</class>
   <extends>QWidget</extends>
   <header>MasterConfigurationPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ServiceConfigurationPage</class>
   <extends>QWidget</extends>
   <header>ServiceConfigurationPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AuthenticationConfigurationPage</class>
   <extends>QWidget</extends>
   <header>AuthenticationConfigurationPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../veyon-configurator.qrc"/>
  <include location="../../core/core.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pageSelector</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>configPages</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutVeyon</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutVeyon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetConfiguration</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetConfiguration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>aboutVeyon()</slot>
  <slot>resetConfiguration()</slot>
 </slots>
</ui>
